sdk/samples/iot/README.md

```diff
@@ -124,7 +124,7 @@ To run the samples, ensure you have the following programs and tools installed o
     cd vcpkg
     git checkout <vcpkg commit> # Checkout the vcpkg commit per vcpkg-commit.txt above.
     ./bootstrap-vcpkg.sh
-    ./vcpkg install --triplet x64-linux curl cmocka paho-mqtt
+    ./vcpkg install --triplet x64-linux curl cmocka paho-mqtt mosquitto
```

Comment:
Do you need libmosquitto-dev and libssl-dev as well? (you might not since you're using vcpkg)

Reply:
The new changes shouldn't use OpenSSL in any way. 
Dependencies of Mosquitto will be addressed by VCPKG.